src/coreclr/jit/emitarm64.cpp

```diff
@@ -9831,16 +9847,59 @@ void emitter::emitIns_R_R_R_R(
             fmt = IF_NONE;
             break;
 
+        case INS_sve_cmpeq:
+        case INS_sve_cmpgt:
+        case INS_sve_cmpge:
+        case INS_sve_cmphi:
+        case INS_sve_cmphs:
+        case INS_sve_cmpne:
+        case INS_sve_cmple:
+        case INS_sve_cmplo:
+        case INS_sve_cmpls:
+        case INS_sve_cmplt:
+            assert(insOptsScalableSimple(opt));
+            assert(isPredicateRegister(reg1));    // DDDD
+            assert(isLowPredicateRegister(reg2)); // ggg
+            assert(isVectorRegister(reg3));       // mmmmm
+            assert(isVectorRegister(reg4));       // nnnnn
+            assert(isScalableVectorSize(attr));   // xx
+            fmt = IF_SVE_CX_4A;
+            break;
+
         default:
             unreached();
             break;
     }
     assert(fmt != IF_NONE);
 
+    // Use aliases.
```

Comment:
any reason why we want to use alias here?

Reply:
We use aliases are they are preferred in disassembly. This also helps to ensure that the assembly from clr matches to that from capstone.